Sources/ObservableStore/ObservableStore.swift

```diff
@@ -175,17 +204,43 @@ public protocol StoreProtocol {
 public final class Store<Model>: ObservableObject, StoreProtocol
 where Model: ModelProtocol
 {
-    /// Stores cancellables by ID
-    private(set) var cancellables: [UUID: AnyCancellable] = [:]
+    private var cancelTransactions: AnyCancellable?
```

Comment:
Is this being used anywhere?

Reply:
Combine subscribers automatically cancel when their cancellable is released. So you have to hold on to the cancellable to keep the subscriber alive. We hold on to the cancellable within the instance so that its lifetime matches the store lifetime. The subscriber will continue receiving as long as the store instance exists.